frontend/products/subscriptions.js

```diff
@@ -0,0 +1,35 @@
+import { appDidStart$, redirects, parseObjectToQueryString } from '@shopgate/engage/core';
+import { CATEGORY_PATTERN } from '@shopgate/engage/category';
+import { SEARCH_PATTERN } from '@shopgate/engage/search';
+import config from '../config';
+
+const { defaultSortOption } = config;
+
+/**
+ * Subscriptions
+ * @param {Function} subscribe subscribe
+ */
+export default (subscribe) => {
```

Comment:
maybe it would be better to add config for this to themes

Reply:
Since its custom sort options, themes do not know what it will be

Reply:
valid point. So we should wait until we migrated this ext to CURB. Then we can use the settings there as default here + you can overide it with setting from here